ethcore/light/Cargo.toml

```diff
@@ -36,10 +36,13 @@ triehash = { path = "../../util/triehash" }
 kvdb = { path = "../../util/kvdb" }
 memory-cache = { path = "../../util/memory_cache" }
 error-chain = { version = "0.12", default-features = false }
+util-error = { path = "../../util/error" }
```

Comment:
This was crate was removed https://github.com/paritytech/parity/pull/9054, do you need it?

Please rebase/merge to master